packages/ubuntu_provision/lib/src/eula/eula_page.dart

```diff
@@ -43,7 +47,11 @@ class _EulaPageState extends ConsumerState<EulaPage> {
           Logger('eula')
               .error('Error loading EULA file: $error', error, stackTrace);
           return _EulaPdfViewer(
-              path: File('/usr/share/desktop-provision/eula/EULA.pdf').path);
+              path: File(p.join(
+                      Platform.environment['DESKTOP_PROVISION_PATH'] ??
+                          defaultFilePath,
+                      'eula/EULA.pdf'))
+                  .path);
```

Comment:
```suggestion
                      Platform.environment['DESKTOP_PROVISION_PATH'] ??
                          defaultFilePath,
                      'eula/EULA.pdf',),)
                  .path,);
```
Add trailing commas, needs formatting afterwards